### ISSUE TITLE
Deal with npm resolution errors

### ISSUE THREAD

**Issue by albertyw** [albertyw/req-update#72]
```
req-update
Updating npm packages
Update npm packages
Updating dependency: @types/react
Update @types/react package to ^18.0.0
Updating dependency: @types/react-dom
Update @types/react-dom package to ^18.0.0
Updating dependency: react
Update react package to ^18.0.0
Updating dependency: react-dom

npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: owloid@1.0.0
npm WARN Found: react-dom@17.0.2
npm WARN node_modules/react-dom
npm WARN   react-dom@"^18.0.0" from the root project
npm WARN   1 more (@excalidraw/excalidraw)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^17.0.2" from @excalidraw/excalidraw@0.11.0
npm WARN node_modules/@excalidraw/excalidraw
npm WARN   @excalidraw/excalidraw@"latest" from the root project
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @excalidraw/excalidraw@0.11.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"^18.0.0" from the root project
npm ERR!   peer react@">= 16" from react-scripts@5.0.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@"^5.0.0" from the root project
npm ERR!   1 more (react-dom)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from @excalidraw/excalidraw@0.11.0
npm ERR! node_modules/@excalidraw/excalidraw
npm ERR!   @excalidraw/excalidraw@"latest" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^17.0.2" from @excalidraw/excalidraw@0.11.0
npm ERR!   node_modules/@excalidraw/excalidraw
npm ERR!     @excalidraw/excalidraw@"latest" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/albertyw/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/albertyw/.npm/_logs/2022-06-11T05_36_20_021Z-debug-0.log
```

**Comment by albertyw** [comment]
```

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: codemancer@1.0.0
npm ERR! Found: mocha@10.0.0
npm ERR! node_modules/mocha
npm ERR!   dev mocha@"^10.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mocha@"^3.X.X || ^4.X.X || ^5.X.X || ^6.X.X || ^7.X.X || ^8.X.X || ^9.X.X" from ts-mocha@9.0.2
npm ERR! node_modules/ts-mocha
npm ERR!   dev ts-mocha@"^9.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/albertyw/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/albertyw/.npm/_logs/2022-06-18T04_45_39_831Z-debug-0.log
```

**Comment by albertyw** [comment]
Fixed in 1094133
